client/site-profiler/components/heading-information/index.tsx

```diff
@@ -50,7 +50,7 @@ export default function HeadingInformation( props: Props ) {
 
 	const onMigrateSite = () => {
 		recordCtaEvent( 'migrateSite' );
-		page( `/setup/import-hosted-site?from=${ domain }` );
+		page( `/setup/hosted-site-migration?ref=site-profiler&from=${ domain }` );
```

Comment:
This path redirects to `/setup/hosted-site-migration/site-migration-identify` which seems like an importer flow.

![CleanShot 2024-05-29 at 12 05 49@2x](https://github.com/Automattic/wp-calypso/assets/12430020/8f14de9e-96d1-42ff-820c-858187a61987)


Should we use `/move` instead?

Reply:
@cpapazoglou, we're using the same underlying components, but this is a migration flow. 😁